assets/stylesheets/bootstrap/_variables.scss

```diff
@@ -574,13 +584,13 @@ $label-default-bg:            $gray-light !default;
 //** Primary label background color
 $label-primary-bg:            $brand-primary !default;
 //** Success label background color
-$label-success-bg:            $brand-success !default;
+$label-success-bg:            #a0cfbb !default;
```

Comment:
These three colours aren't the product of a consistent transformation on variables like `$brand-success`